src/targets/chrome/docker.js

```diff
@@ -180,7 +190,8 @@ function createChromeDockerTarget({
 
   process.on('SIGINT', () => {
     if (dockerId) {
-      execSync(`${dockerPath} kill ${dockerId}`);
+      const maybeSudo = dockerAsSudo ? 'sudo' : '';
+      execSync(`${maybeSudo}${dockerPath} docker kill ${dockerId}`);
```

Comment:
This looks wrong to me. Shouldn't it be `execSync(`${maybeSudo} ${dockerPath} kill ${dockerId}`);`?